src/commands/run.js

```diff
@@ -0,0 +1,89 @@
+const log = require('@dhis2/cli-helpers-engine').reporter
+const spawn = require('cross-spawn')
+const fs = require('fs-extra')
+const path = require('path')
+
+module.exports.command = 'run <package> <codemod> <files..>'
+module.exports.alias = 'r'
+module.exports.desc = `* Runs a jscodemod.
+* Packages must be in the "node_modules" dir
+* Codemods must be located in "codemods" folders
+
+Example:
+* run @dhis2/cli-utils-cypress name-of-a-codemod cypress/integration/**/*.js --forwardArgs key1=val1 key2=val2
+-> Will run the "name-of-a-codemod.js"
+-> Located in ./node_modules/@dhis2/cli-utils-cypress/codemods/name-of-a-codemod.js
+-> Will apply codemod to all js files located in cypress/integration
+-> Will forward { key1: 'val1', key2: 'val2' } to the codemod
+`
+
+module.exports.builder = yargs =>
+    yargs
+        .positional('package', {
+            describe: 'The name of the package that contains the codemod',
+            type: 'string',
+        })
+        .positional('codemod', {
+            describe: 'Name of the codemod file',
+            type: 'string',
+        })
+        .positional('files', {
+            describe: 'Files or path the codemod should be applied to',
+            type: 'string',
+        })
+        .option('package-folder', {
+            describe: 'The package the folder can be found in',
+            type: 'string',
+            default: path.join(`${process.cwd()}`, 'node_modules')
+        })
+        .option('codemods-folder', {
+            describe: 'The package the codemods can be found in',
+            type: 'string',
+            default: 'codemods',
+        })
+        .option('forward-args', {
+            describe: 'Args that should be forwarded to the codemod',
+            type: 'array',
+            default: [],
+        })
+
+module.exports.handler = argv => {
+    const {
+        codemod,
+        codemodsFolder,
+        files,
+        forwardArgs,
+        package,
+        packageFolder,
+    } = argv
+
+    const packagePath = path.join(packageFolder, package)
+    if (!fs.pathExistsSync(packagePath)) {
+        log.error(`The package "${package}" is not installed. Check your package.json if it's included.`)
+        process.exit(1)
+    }
+
+    const codemodsPath = path.join(packagePath, codemodsFolder)
+    if (!fs.pathExistsSync(codemodsPath)) {
+        log.error(`The package "${package}" does not contain any codemods.`)
+        process.exit(1)
+    }
+
+    const scriptPath = path.join(codemodsPath, `${codemod}.js`)
+    if (!fs.pathExistsSync(scriptPath)) {
+        log.error(`The package "${package}" does not have a codemod called "${codemod}"`)
+    }
+
+    log.info('Starting running the codemod\n')
+
+    const result = spawn.sync('npx', [
```

Comment:
Any reason to use `cross-spawn` instead of `import { exec } from '@dhis2/cli-helpers-engine'`

Reply:
Just copied that from the cypress repo, didn't know we had an alternative in our helpers engine

Reply:
I think we should use [`cross-spawn`](https://github.com/moxystudio/node-cross-spawn), but @amcgee is right, we should use the helpers-engine, and instead move `cross-spawn` there so all our CLI libs get the benefits.

Not only does `cross-spawn` work better on Windows, it also solves some problem Node has with posix relative paths and command shims in `node_modules/.bin` which e.g. `cli-style` (and this tool) relies heavily on.

Reply:
I'm game for using `cross-spawn`, but we should put it in the helper function so it's used by all tools rather than just some.

Reply:
Not using cross-spawn anymore, see other comment below